broadword.hpp

```diff
@@ -169,7 +169,7 @@ namespace succinct { namespace broadword {
         if (!x) {
             return false;
         }
-        ret = bit_position(x & -int64_t(x));
+        ret = bit_position(x & static_cast<uint64_t>(-int64_t(x)));
```

Comment:
Yes, you are right. Actually I realized that _signed int overflow_ is UB in C++. It definitely makes sense to change it to `x & -x ` and completely skip the conversion to `int64_t`